packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize } from '@material-ui/core/utils';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = (theme) => ({
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    alignSelf: 'baseline',
+    borderStyle: 'solid',
+    borderWidth: 2,
+    flex: 0,
+    padding: 4,
+    borderRadius: 999,
+    // marginRight: 'auto',
+    // marginLeft: 'auto',
+    top: 8,
+    position: 'relative',
+    boxShadow: theme.shadows[2],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="default"`. */
+  defaultDefault: {
+    borderColor: theme.palette.grey[300],
+    backgroundColor: theme.palette.grey[300],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="outlined"`. */
+  outlinedDefalut: {
+    borderColor: theme.palette.grey[300],
+    backgroundColor: 'white', // TODO: fix color white
+  },
+  /* Styles applied when the component is passed children. */
+  withChildren: {
+    marginTop: -8,
+  },
+  /* Styles applied to the root element if `color="primary"` and `variant="default"`. */
+  defaultPrimary: {
+    color: theme.palette.primary.contrastText,
+    backgroundColor: theme.palette.primary.main,
+    borderColor: theme.palette.primary.main,
+  },
+  /* Styles applied to the root element if `color="primary"` and `variant="outlined"`. */
+  outlinedPrimary: {
+    backgroundColor: 'white',
+    borderColor: theme.palette.primary.main,
+  },
+  /* Styles applied to the root element if `color="secondary"` and `variant="default"`. */
+  defaultSecondary: {
+    color: theme.palette.secondary.contrastText,
+    backgroundColor: theme.palette.secondary.main,
+    borderColor: theme.palette.secondary.main,
+  },
+  /* Styles applied to the root element if `color="secondary"` and `variant="outlined"`. */
+  outlinedSecondary: {
+    backgroundColor: 'white',
+    borderColor: theme.palette.secondary.main,
+  },
+});
+
+const TimelineItemDot = React.forwardRef(function TimelineItemDot(props, ref) {
+  const {
+    classes,
+    className,
+    component: Component = 'span',
+    color = 'default',
```

Comment:
We are aiming toward removing all the `default` occurrences of the color: #13028.
Maybe we should rename the color `grey`?

```suggestion
    color = 'default',
```